companies/bfs-health-finance.json

```diff
@@ -1,24 +1,32 @@
 {
     "slug": "bfs-health-finance",
+    "relevant-countries": [
+        "de"
+    ],
     "categories": [
-        "credit agency"
+        "credit agency",
+        "health"
     ],
     "name": "BFS health finance GmbH",
-    "address": "Carl-Bertelsmann-Straße 270\n33311 Gütersloh\nDeutschland",
-    "phone": "+49 800 447 32 54",
-    "fax": "+49 231 945 362 888",
-    "email": "datenschutz@meinebfs.de",
```

Comment:
I couldn't find a source for this. :/

Reply:
They do still have a press release mentioning the email on their site (https://meinebfs.de/dsgvo-die-datenschutzgrundverordnung/) but it does seem like there was some restructuring and they are not using that email anymore, unfortunately.